frontend/src/components/Meal/MealContainer.js

```diff
@@ -1,29 +1,53 @@
-import React from 'react';
-import { Meal } from "./Meal";
-import { MealForm } from "./MealForm";
-import { makeStyles } from '@material-ui/core/styles';
+import React, { useEffect } from "react";
+import Button from "@material-ui/core/Button";
+import ButtonGroup from '@material-ui/core/ButtonGroup';
+
+import { Day } from "./Day";
+import { TRACKED_DAYS_THRESHOLD } from "../../config"
+import { useAppStyles }  from "../../styles/theme";
 
-const useStyles = makeStyles({
-    mealContainer: {
-      padding: 30,
-    },
-  });
 
 const MealContainer = () => {
-    const classes = useStyles();
+  const classes = useAppStyles();
+  const range = Array.from(Array(TRACKED_DAYS_THRESHOLD).keys());
```

Comment:
Please create a function called:

`const generateRange = from => Array.from(Array.from(from).keys());`

Then you can use this function here (for initial range), and then later, in useEffect https://github.com/soniasonia/meal-tracker/pull/6/files#diff-ff75b0686171e9baf4c3219eca5cd0b97a9429ddf30c8764cdfaea3b4a2e9be8R17 - that way you'll be sure to use same code. It also makes it more explicit in case someone wonders what it's doing ;)

Reply:
OK